clear-intermediate-datasets/python-runnables/clear_int_datasets/runnable.py

```diff
@@ -78,11 +79,14 @@ def run(self, progress_callback):
                                                                 str(flow_outputs)))
         
         # Identify standalone, intermediate, and partitioned datasets
+        excluded_datasets = []
         standalone_datasets = []
         intermediate_datasets = []
         partitioned_datasets = []
 
         for dataset in all_datasets:
+            if dataset["name"] in manually_selected_datasets:
```

Comment:
Another general remark : Prefer using variables to diminish computation, e.g dataset_name = dataset["name"]